js/Track/library/dbSNP.js

```diff
@@ -1,12 +1,14 @@
-Genoverse.Track.dbSNP = Genoverse.Track.extend({
+const { Track } = require('../../Track')
+
+module.exports = Track.extend({
   id               : 'dbSNP',
   name             : 'dbSNP',
   info             : 'All sequence variants from the database of Single Nucleotide Polymorphisms (dbSNP)',
   url              : '//rest.ensembl.org/overlap/region/human/__CHR__:__START__-__END__?feature=variation;content-type=application/json',
   dataRequestLimit : 5000000, // As per e! REST API restrictions
   threshold        : 1e5,
   labels           : false,
-  legend           : true,
+  // legend           : true,
```

Comment:
Should this be added back in? I think it was from testing the circular dependency?

```suggestion
  legend           : true,
```